components/kyma-environment-broker/internal/process/provisioning/create_runtime.go

```diff
@@ -97,7 +84,7 @@ func (s *CreateRuntimeStep) Run(operation internal.ProvisioningOperation, log lo
 		return operation, 1 * time.Minute, nil
 	}
 
-	err = s.instanceStorage.Insert(internal.Instance{
+	err = s.instanceStorage.Update(internal.Instance{
```

Comment:
it is better to get instance, update field(s) which must be updated and save it (operation update)